servers/xr/xr_vrs.cpp

```diff
@@ -33,6 +33,10 @@
 #include "servers/rendering/renderer_scene_render.h"
 #include "servers/rendering_server.h"
 
+// TODO: This seems to give a result that matches more closely a circle, but is that actually what we want? To discuss and document with Baastian the formula behind the original computation.
```

Comment:
I never had a good answer for this either. From talking to other VR engineers most are in the same boat as us. The few vendors that have thrown serious money at testing different approaches and finding out what works best, don't seem to be sharing those findings (or I am just not looking in the right places, totally possible:) ). 

I thus also have no opinion on which is best. I'm happy to switch to the more rounded approach.

Reply:
We should switch as it leads to better generation of the tile distribution on hardware. As it is, it wasn't generating bigger tiles far away from the focal point because along each cardinal direction, the original generation always puts it at max density.

While it does leave the functionality a bit unused, we can clarify with a comment that we leave the door open to future improvements and go with a more boring approach to give the best result for a general use case scenario.

Reply:
I'm more than happy with this take, so I'm all for making it the only way we build these maps.